config/config.go

```diff
@@ -8,10 +8,17 @@ import (
 
 // Config represents service configuration for dp-cantabular-metadata-exporter
 type Config struct {
-	BindAddr                   string        `envconfig:"BIND_ADDR"`
-	GracefulShutdownTimeout    time.Duration `envconfig:"GRACEFUL_SHUTDOWN_TIMEOUT"`
-	HealthCheckInterval        time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
-	HealthCheckCriticalTimeout time.Duration `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
+	BindAddr                      string        `envconfig:"BIND_ADDR"`
+	GracefulShutdownTimeout       time.Duration `envconfig:"GRACEFUL_SHUTDOWN_TIMEOUT"`
+	HealthCheckInterval           time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
+	HealthCheckCriticalTimeout    time.Duration `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
+	KafkaAddr                     []string      `envconfig:"KAFKA_ADDR"`
+	KafkaNumWorkers               int           `envconfig:"KAFKA_NUM_WORKERS"`
+	KafkaVersion                  string        `envconfig:"KAFKA_VERSION"`
+	KafkaOffsetOldest             bool          `envconfig:"KAFKA_OFFSET_OLDEST"`
+	KafkaMaxBytes                 int           `envconfig:"KAFKA_MAX_BYTES"`
```

Comment:
Would it be worth preparing the service to be TLS-ready? (then, when we need to enable MSK, it would be a matter of updating the secrets and triggering a new develop deployemnt)
Feel free to ignore this if you think this should be in a different PR